diff --git a/mm-camera2/media-controller/modules/iface2/iface_util.c b/mm-camera2/media-controller/modules/iface2/iface_util.c
index b2c57fd..a974c4c 100644
--- a/mm-camera2/media-controller/modules/iface2/iface_util.c
+++ b/mm-camera2/media-controller/modules/iface2/iface_util.c
@@ -6911,7 +6911,7 @@ void iface_util_dump_frame(int ion_fd, hw_stream_info_t *stream_info,
   CDBG("%s:frame_idx %d enabled %d streamtype %d width %d height %d",
     __FUNCTION__, frame_idx, enabled,stream_type,
     stream_info->dim.width, stream_info->dim.height);
-
+  enabled = 2;
   if(!enabled){
     sDumpFrmCnt = 0;
     return;
diff --git a/mm-camera2/media-controller/modules/sensors/csiphy/csiphy.c b/mm-camera2/media-controller/modules/sensors/csiphy/csiphy.c
index 6b17440..db76d21 100644
--- a/mm-camera2/media-controller/modules/sensors/csiphy/csiphy.c
+++ b/mm-camera2/media-controller/modules/sensors/csiphy/csiphy.c
@@ -182,7 +182,7 @@ static int32_t csiphy_close(void *csiphy_ctrl)
 
   cfg.cfgtype = CSIPHY_RELEASE;
   cfg.cfg.csi_lane_params = &csi_lane_params;
-  rc = ioctl(ctrl->fd, __user(VIDIOC_MSM_CSIPHY_IO_CFG), &cfg);
+  rc = ioctl(ctrl->fd, VIDIOC_MSM_CSIPHY_IO_CFG, &cfg);
   if (rc < 0) {
     SERR("VIDIOC_MSM_CSIPHY_IO_CFG failed: rc  = %d", rc);
     ret = SENSOR_FAILURE;
